From a5afdd61f6c8c129a57ff20190ea95e1d2903da6 Mon Sep 17 00:00:00 2001
From: Johannes Thoma <johannes@johannesthoma.com>
Date: Fri, 12 Nov 2021 18:59:37 +0100
Subject: [PATCH 6/6] drbd_transport_tcp: long -> LONG_PTR

long is 32 bit on Windows (also on 64 bit systems)
and should not be used at all.
---
 drbd/drbd_transport_tcp.c | 4 ++--
 1 file changed, 2 insertions(+), 2 deletions(-)

diff --git a/drbd/drbd_transport_tcp.c b/drbd/drbd_transport_tcp.c
index 2df2e56..76578ab 100644
--- a/drbd/drbd_transport_tcp.c
+++ b/drbd/drbd_transport_tcp.c
@@ -76,7 +76,7 @@ static int dtt_recv_pages(struct drbd_transport *transport, struct drbd_page_cha
 static void dtt_stats(struct drbd_transport *transport, struct drbd_transport_stats *stats);
 static void dtt_net_conf_change(struct drbd_transport *transport, struct net_conf *new_net_conf);
 static void dtt_set_rcvtimeo(struct drbd_transport *transport, enum drbd_stream stream, LONG_PTR timeout);
-static long dtt_get_rcvtimeo(struct drbd_transport *transport, enum drbd_stream stream);
+static LONG_PTR dtt_get_rcvtimeo(struct drbd_transport *transport, enum drbd_stream stream);
 static int dtt_send_page(struct drbd_transport *transport, enum drbd_stream, struct page *page,
 		int offset, size_t size, unsigned msg_flags);
 static int dtt_send_zc_bio(struct drbd_transport *, struct bio *bio);
@@ -1193,7 +1193,7 @@ static void dtt_set_rcvtimeo(struct drbd_transport *transport, enum drbd_stream
 	socket->sk->sk_rcvtimeo = timeout;
 }
 
-static long dtt_get_rcvtimeo(struct drbd_transport *transport, enum drbd_stream stream)
+static LONG_PTR dtt_get_rcvtimeo(struct drbd_transport *transport, enum drbd_stream stream)
 {
 	struct drbd_tcp_transport *tcp_transport =
 		container_of(transport, struct drbd_tcp_transport, transport);
-- 
2.17.0

